### PR TITLE
Cleanup TriAnalyzer docs

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: lib/matplotlib/tri/tritools.py [hunk 1]
```diff
@@ -61,22 +54,21 @@ def circle_ratios(self, rescale=True):
 
         To avoid unduly low values due to a difference of scale between the 2
         axis, the triangular mesh can first be rescaled to fit inside a unit
-        square with :attr:`scale_factors` (Only if *rescale* is True, which is
```
Comment (anntzer) [inline lib/matplotlib/tri/tritools.py]: well, it's an :attr:, no?
(what are we doing about them, btw?)

File: lib/matplotlib/tri/tritools.py [hunk 2]
```diff
@@ -61,22 +54,21 @@ def circle_ratios(self, rescale=True):
...
+        square with `scale_factors` (Only if *rescale* is True, which is
         its default value).
 
         Parameters
         ----------
         rescale : bool, default: True
             If True, a rescaling will be internally performed (based on
-            :attr:`scale_factors`, so that the (unmasked) triangles fit
```
Comment (anntzer) [inline lib/matplotlib/tri/tritools.py]: same

File: lib/matplotlib/tri/tritools.py [hunk 3]
```diff
@@ -12,18 +12,13 @@ class TriAnalyzer:
...
 
     Parameters
     ----------
-    triangulation : :class:`~matplotlib.tri.Triangulation` object
+    triangulation : `~matplotlib.tri.Triangulation`
         The encapsulated triangulation to analyze.
-
-    Attributes
-    ----------
-    `scale_factors`
```
Comment (anntzer) [inline lib/matplotlib/tri/tritools.py]: would be nice if we could get sphinx to like :attr: to here...

Comment (timhoffm) [reply lib/matplotlib/tri/tritools.py]: If not using markup, this links to the property documentation.

I'm not sure if we should list properties as attributes if they are not writable, but attribute handling is for another time. With this change this is already an improvement.

File: lib/matplotlib/tri/tritools.py (hunk 2 above)
Comment (timhoffm) [reply lib/matplotlib/tri/tritools.py]: same (It's actually a property and now links there.)

File: lib/matplotlib/tri/tritools.py (hunk 1 above)
Comment (timhoffm) [reply lib/matplotlib/tri/tritools.py]: It's actually a property and now links there.

File: lib/matplotlib/tri/tritools.py (hunk 3 above)
Comment (anntzer) [reply lib/matplotlib/tri/tritools.py]: ah, that works nicely

**Comment by anntzer** [comment]
can selfmerge postci

**Pull request merged**